roles/internal/grok/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+grok_clone_directory: /opt/grok
+grok_version_tag: v2.3.0
```

Comment:
I would put both of these in the `roles/internal/grok/defaults/main.yml` file, as they are super hard to override where they are. Here is a [good reference](https://codereviewvideos.com/course/ansible-tutorial/video/variable-precedence-where-to-put-your-role-vars) talking about where to put variables. The TL;DR is that you should always use the defaults, not vars.

Reply:
Right, I did have these there and then I stupidly moved the entire file to cause the below Cantaloupe option to get set. I'll fix them both.